src/coord/coord.rs

```diff
@@ -1423,6 +1422,21 @@ where
                                         }),
                                     );
                                 }
+                                SinkConnector::AvroOcf(AvroOcfSinkConnector { path }) => {
+                                    broadcast(
+                                        &mut self.broadcast_tx,
+                                        SequencedCommand::AppendLog(
+                                            MaterializedEvent::AvroOcfSink {
+                                                id: entry.id(),
+                                                path: path
+                                                    .to_str()
+                                                    .unwrap_or("<invalid utf8>")
+                                                    .to_string(),
```

Comment:
I think `path` should be a `Vec<u8>`, so that we can handle this properly.

Reply:
I'm happy to make the changes to do that - but then the output from this log will not be user friendly at all e.g.

```
materialize=> select path from mz_avro_ocf_sinks;
                                                                                                             path
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 \x2f7661722f666f6c646572732f6a702f305f6e797870796a3476786738783463316b783176686c3830303030676e2f542f2e746d707441317a6d642f62617369632d73696e6b2d7531312d313538353936363130312d353830353134363331353038343638313330332e6f6366
(1 row)

materialize=>
```

not sure which is worse?